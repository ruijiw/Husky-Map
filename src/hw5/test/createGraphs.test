# Specification testing file.
# Tests that multiple graphs can be created


# Create the first graph
CreateGraph graph1

# Create the second graph
CreateGraph graph2

# Create the third graph
CreateGraph graph3

# Create the 4th graph
CreateGraph graph4

# Create the 5th graph
CreateGraph graph5

# Create the 6th graph
CreateGraph graph6

# Create the 7th graph
CreateGraph graph7