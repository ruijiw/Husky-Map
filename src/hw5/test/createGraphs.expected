# Specification testing file.
# Tests that multiple graphs can be created


# Create the first graph
created graph graph1

# Create the second graph
created graph graph2

# Create the third graph
created graph graph3

# Create the 4th graph
created graph graph4

# Create the 5th graph
created graph graph5

# Create the 6th graph
created graph graph6

# Create the 7th graph
created graph graph7